app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt

```diff
@@ -34,7 +34,7 @@ object AppConfig {
     val chatBotLink
         get() = getLocalized("chatBotLink")
     val helpMarkdown
-        get() = getLocalized("helpMarkdown")
+        get() = getLocalized("helpMarkdown2")
```

Comment:
`helpMarkdown2` nemá default hodnotu?

Reply:
jj, ještě přidat helpMarkdown2 do default values, vezmi to z helpMarkdown z DEV remote configu